lib/Ed25519Signature2020.js

```diff
@@ -0,0 +1,148 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import jsonld from 'jsonld';
+import jsigs from 'jsonld-signatures';
+const {suites: {LinkedDataSignature}} = jsigs;
+const base58btc = require('base58-universal');
+import {
+  Ed25519VerificationKey2020
+} from '@digitalbazaar/ed25519-verification-key-2020';
+
+export class Ed25519Signature2020 extends LinkedDataSignature {
+  /**
+   * @param {object} options - Options hashmap.
+   * @param {string} [options.verificationMethod] - A key id URL to the paired
+   *   public key.
+   * @param {Function} [options.signer] - Signer function.
+   *
+   * Advanced optional parameters and overrides:
+   *
+   * @param {object} [options.key] - An optional crypto-ld KeyPair.
+   * @param {object} [options.proof] - A JSON-LD document with options to use
+   *   for the `proof` node (e.g. any other custom fields can be provided here
+   *   using a context different from security-v2).
+   * @param {string|Date} [options.date] - Signing date to use if not passed.
+   * @param {boolean} [options.useNativeCanonize] - Whether to use a native
+   *   canonize algorithm.
+   * @param {boolean} [options.compactProof=false] - Recommendation to
+   *   jsonld-signature on whether to compactProof. This is defined per suite,
+   *   and the expectation is that the doc being verified conforms to the
+   *   suite's context.
+   */
+  constructor({
+    signer, verificationMethod, key, proof, date, useNativeCanonize,
+    compactProof = false
```

Comment:
Don't need `compactProof` anymore, it's not a thing with jsigs 8.x+.

Reply:
Yep, removed.